src/js/disability-benefits/526EZ/helpers.jsx

```diff
@@ -283,3 +304,105 @@ export const evidenceSummaryView = ({ formData }) => {
     </div>
   );
 };
+
+const FullNameViewField = ({ formData }) => {
+  const { first, middle, last, suffix } = formData;
+  return <strong>{first} {middle} {last} {suffix}</strong>;
+};
+
+const SsnViewField = ({ formData }) => {
+  const ssn = formData.slice(5);
+  const mask = <span>•••-••-</span>;
+  return <p>Social Security number: {mask}{ssn}</p>;
+};
+
+const VAFileNumberViewField = ({ formData }) => {
+  const vaFileNumber = formData.slice(5);
+  const mask = <span>•••-••-</span>;
+  return <p>VA file number: {mask}{vaFileNumber}</p>;
+};
+
+const DateOfBirthViewField = ({ formData }) => {
+  return <p>Date of birth: <DateWidget value={formData} options={{ monthYear: false }}/></p>;
+};
+
+const GenderViewField = ({ formData }) => <p>Gender: {genderLabels[formData]}</p>;
+
+export const veteranInformationViewField = ({ formData }) => {
+  return (
+    <div>
+      <FullNameViewField formData={formData.fullName}/>
+      <SsnViewField formData={formData.socialSecurityNumber}/>
+      <VAFileNumberViewField formData={formData.vaFileNumber}/>
+      <GenderViewField formData={formData.gender}/>
+      <DateOfBirthViewField formData={formData.dateOfBirth}/>
+    </div>
+  );
+};
+
+const lowerCaseFirstLetter = (word) => word[0].toLowerCase().concat(word.slice(1));
+
+const lowerCaseAll = (words) => words.map(word => lowerCaseFirstLetter(word));
+
+const camelize = (words) => {
+  const word = words[0];
+  const rest = words.slice(1).join('');
+  return lowerCaseFirstLetter(word).concat(rest);
+};
+
+const kebabize = (words) => {
+  return lowerCaseAll(words).join('-');
+};
+
+const getVerifiedPairPageNames = (chapterTitleWords) => {
+  const unverifiedPageName = camelize(chapterTitleWords);
+  chapterTitleWords.unshift('review');
+  const verifiedPageName = camelize(chapterTitleWords);
+  return { unverifiedPageName, verifiedPageName };
+};
+
+const getVerifiedPairPagePaths = (chapterTitleWords) => {
+  const unverifiedPagePath = kebabize(chapterTitleWords);
+  chapterTitleWords.unshift('review');
+  const verifiedPagePath = kebabize(chapterTitleWords);
+  return { unverifiedPagePath, verifiedPagePath };
+};
+
+export const getVerifiedChapterPair = (chapterConfig) => {
```

Comment:
I think I understand what this function is doing, but the word "pair" makes me think I'm missing something. You might consider adding some comments around that.

Reply:
Good point, it should be updated to getChapter (pair will be removed throughout). `createVeteranInfoChapter`should be curried so that `isReview` doesn't need to be supplied. Will update.